docker-compose/monitor/README.md

```diff
@@ -42,12 +44,37 @@ docker compose down
 **Warning:** The included [docker-compose.yml](./docker-compose.yml) file uses the `latest` version of Jaeger and other components. If your local Docker registry already contains older versions, which may still be tagged as `latest`, you may want to delete those images before running the full set, to ensure consistent behavior:
 
 ```bash
-docker rmi -f jaegertracing/all-in-one:latest
-docker rmi -f otel/opentelemetry-collector-contrib:latest
-docker rmi -f prom/prometheus:latest
-docker rmi -f grafana/grafana:latest
+make clean-all
+```
+
+## Development
+
+These steps allow for running the system necessary for SPM, built from Jaeger's source.
+
+The primary use case is for testing source code changes to the SPM feature locally.
+
+### Build jaeger-all-in-one docker image
+
+```shell
+make build
+```
+
+## Bring up the dev environment
+
+```bash
+make run-dev
+```
+
+## Backwards compatibility testing with spanmetrics processor
+
+```bash
+make run-dev-processor
 ```
 
+For each "run" make target, you should expect to see the following in the Monitor tab after a few minutes:
+
+![Monitor Screenshot](https://github.com/jaegertracing/jaeger/assets/26584478/f0fd5d26-c486-4de6-892b-ac48c3e8e425)
```

Comment:
how is this link determined? Can we put it under `images/` instead?

Reply:
Good idea, updated in [f030268](https://github.com/jaegertracing/jaeger/pull/4452/commits/f030268105324ef71b6e89201e55c5959d732848).